c-dependencies/js-compute-runtime/xqd-world/xqd_world.h

```diff
@@ -65,11 +65,10 @@ typedef struct {
 
 typedef uint8_t fastly_http_cache_override_tag_t;
 
-#define FASTLY_HTTP_CACHE_OVERRIDE_TAG_NONE (1 << 0)
```

Comment:
@guybedford @elliottt - this is one of the more important parts of this PR - when we moved to the component model .wit definitions, we accidentally got an off-by-one error for all our cache-override flag values

Reply:
This file is generated by `wit-bindgen`, is there any way we can make the fix in `xqd.wit` instead so that it generates the correct output? Perhaps just removing the `none` variant?

Reply:
Looks like you removed the constructor in xqd.wit already, so this looks good to me!